grass7/raster/r.replace/r.replace.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
+#
+############################################################################
+#
+# MODULE:    r.replace
+# AUTHOR(S): Stefano Gobbi and Paolo Zatelli
+# PURPOSE:   Replaces the values of pixels of a given category with values
+#            of the surrounding pixels
+#
+# COPYRIGHT: (C) 2019 by Stefano Gobbi and Paolo Zatelli
+#
+#   This program is free software under the GNU General Public
+#   License (>=v2). Read the file COPYING that comes with GRASS
+#   for details.
+#
+#############################################################################
+#%Module
+#% description: Replaces the values of pixels of a given category with values of the surrounding pixels.
+#% keyword: raster
+#% keyword: replace
```

Comment:
Please use "algebra" as the second keyword (the "topics" manual page [1] is generated from the second keyword)

[1] https://grass.osgeo.org/grass77/manuals/topic_algebra.html